src/tqec/templates/orchestrator.py

```diff
@@ -313,6 +315,7 @@ def _build_array(self, indices_map: tuple[int, ...]) -> numpy.ndarray:
         shape = self._get_shape_from_bounding_box(bbul, bbbr)
 
         ret = numpy.zeros(shape.to_numpy_shape(), dtype=int)
+        increments = numpy.zeros(shape.to_numpy_shape(), dtype=object)
```

Comment:
As increments are ultimately 2-dimensional tuples, it might be more "numpy" to have a 
```suggestion
        increments = numpy.zeros((*shape.to_numpy_shape(), 2), dtype=int)
```
and to accordingly change the other use of this array.

Reply:
I wasn't sure what's "pythonic", I can do this though

Reply:
Let's maybe wait for another opinion on that. On my side, I always expect `numpy` arrays to contain native numbers (complex, float and int, mainly), which is the reason why I made this comment.

Reply:
I went with yours because I think its more efficient. The intention is a little less clear, but I think that's fine